src/app/validation.js

```diff
@@ -0,0 +1,23 @@
+
+/**
+ * @param {string} input
+ */
+
+export default (input) => {
+  const value = parseInt(input, 10)
```

Comment:
existem algumas desvantagens em usar `parseInt` ao invés de `Number`:
https://stackoverflow.com/questions/1133770/convert-a-string-to-an-integer

Reply:
Refatorado.